internal/k8s/env.go

```diff
@@ -42,6 +43,8 @@ func EnvFromString(s string) Env {
 		return EnvDockerDesktop
 	} else if Env(s) == EnvMicroK8s {
 		return EnvMicroK8s
+	} else if Env(s) == "kubernetes-admin@kind" {
```

Comment:
can you add this to the test table?

fwiw, i've also seen this as kubernetes-admin@kind-1, so it might be worth doing a prefix check or smth

Reply:
d'oh, yes! Will do